game/Player.h

```diff
@@ -0,0 +1,25 @@
+enum class Character {NOT_SELECTED, Alice, Bob, Carol, Dave};
+
+class Player {
+    private:
+        int id;
+        Character character;
+        float speed;
+        int insecurity;
```

Comment:
I think each player will only have one type of level. Would it be better to have only one "level" variable to represent both?

Reply:
Updated 👍